src/test/java/kg/jarkyn/server/FileHelpers.java

```diff
@@ -0,0 +1,39 @@
+package kg.jarkyn.server;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileHelpers {
+    private static String projectFolderPath = System.getProperty("user.dir");
+    public static String publicFolderPath = projectFolderPath + "/src/test/resources/public";
+    public static String publicFolderRequestPath = "/";
+    static String plainFileName = "file1";
```

Comment:
I see. It's just my opinion, and I'm aware you'll probably change this, but I kind of feel like exposing package variables/methods just for testability is a bit of a design smell. I see this technique used quite often, and I can see it being justified when working on legacy code where there are few tests.  However I'd always recommend striving to test code through the public API/methods.